drizzle-orm/src/postgres-js/driver.ts

```diff
@@ -20,6 +20,14 @@ export function drizzle<TSchema extends Record<string, unknown> = Record<string,
 	client: Sql,
 	config: DrizzleConfig<TSchema> = {},
 ): PostgresJsDatabase<TSchema> {
+	const transparentParser = (val: any) => val;
+
+	// Override postgres.js default date parsers: https://github.com/porsager/postgres/discussions/761
+	for (const type of ['1184', '1082', '1083', '1114']) {
```

Comment:
Wouldn't adding this setter for both parsers and serializers resolve the issues without altering the mapping in the timestamp column class? It seems like it should always return string from database in this case, as long as you just using 
```ts 
const transparentParser = (val: any) => val;
```

Reply:
Yes, you are correct. But I thought that handling what's received from the driver without assuming what it is might be smarter/shielded from future similar bugs.
Let me know if you'd like me to revert the mappings I added at `timestamp.ts`